packages/scrollable-image/src/components/with-waypoints.js

```diff
@@ -55,6 +75,7 @@ const withWaypoints = WrappedComponent => {
       // top boundary leave from viewport
       if (previousPosition === Waypoint.inside) {
         if (currentPosition === Waypoint.above) {
+          if (bottomBoundaryPosition === Waypoint.inside) return
```

Comment:
Why do we need to know `bottomBoundaryPosition` here?

Reply:
Please see [this commit](https://github.com/twreporter/orangutan-monorepo/pull/84#issuecomment-737207322).

I try to explain more detail about my thought about this issue and this idea in it.

Reply:
According to the comments [here](https://github.com/taylrj/orangutan-monorepo/pull/2#issuecomment-737886324) and [here](https://github.com/taylrj/orangutan-monorepo/pull/2#issuecomment-749332320), we will move on with current solution for rapid scrolling.

Reply:
To be more understandable, I will append comments to explain it.